package.json

```diff
@@ -34,7 +34,8 @@
     "build:svgs:index": "create-index src/svgs/ --recursive --update --extensions svg",
     "publish-package": "run-s build && yarn publish",
     "heroku-postbuild": "yarn run build:icons && yarn run build:icons:index && yarn run build-storybook",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "chromatic": "chromatic --skip '@(master|dependabot/**)' --project-token"
```

Comment:
We want to run chromatic on feature branches *prior* to being merged into master. When `master` rebuilds we do not want to trigger another build on chromatic. 

I've also set it up to ignore dependabot PRs for now. Once we settle on billing (which might be free, since the project is open source), we can adjust this so we can rely on Chromatic to help us sanity-check dependency updates as well. 

I previously [included the `--exit-on-upload` flag](https://github.com/curology/radiance-ui/pull/409/commits/f112b6e2e31e115c0a1211586fe2e5092d3064bb) here as well, but for whatever reason all subsequent commits after [changing the build step order in CI](https://github.com/curology/radiance-ui/pull/409/commits/fe8513d3c09b6f4e04be58d7950b7a5a9f2f1f4f) has meant we haven't triggered it, so for now we will have to rely on the `build` CI step itself failing if there are visual regressions as our check, instead of the Chromatic-specific reporter.